tools/tpm2_encryptdecrypt.c

```diff
@@ -326,8 +326,7 @@ tool_rc tpm2_tool_onrun(ESYS_CONTEXT *ectx, tpm2_option_flags flags) {
     tool_rc rc = tpm2_util_object_load_auth(ectx, ctx.encryption_key.ctx_path,
         ctx.encryption_key.auth_str, &ctx.encryption_key.object, false);
     if (rc != tool_rc_success) {
-        LOG_ERR("Invalid object key authorization, got\"%s\"",
-            ctx.encryption_key.auth_str);
+        LOG_ERR("Invalid object key authorization");
```

Comment:
we can probably just delete all these LOG lines, since the call to tpm2_util_object_load_auth() should just LOG all the errors. I would just delete them all, and make sure the routine logs it. This will remove duplicate code from the tools and ensure error messages stay consistent.

Reply:
Applies here and throughout the tools code.

Reply:
I guess though for things that take multiple objects the exact object that got the wrong password isn't known. N/M